stage/configuration/test_directory_origin.py

```diff
@@ -633,11 +633,74 @@ def test_directory_origin_configuration_include_custom_delimiter(sdc_builder, sd
+                                <prc:price>29.99</prc:price>
+                            </b:book>
+                            <b:book xmlns:b="http://books.com/book">
+                                <title lang="en_us">Learning XML</title>
+                                <prc:price>39.95</prc:price>
+                            </b:book>
+                        </bookstore>"""
+    try:
+        logger.debug('Creating files directory %s ...', files_directory)
+        shell_executor(f'mkdir {files_directory}')
+        file_writer(os.path.join(files_directory, FILE_NAME), FILE_CONTENTS)
+
+        pipeline_builder = sdc_builder.get_pipeline_builder()
+        directory = pipeline_builder.add_stage('Directory')
+        directory.set_attributes(data_format=data_format,
+                                 files_directory=files_directory,
+                                 file_name_pattern='xml_include_xpath_file*',
+                                 file_name_pattern_mode='GLOB',
+                                 delimiter_element='/*[1]/*',
+                                 include_field_xpaths=include_field_xpaths)
+        trash = pipeline_builder.add_stage('Trash')
+        directory >> trash
+        pipeline = pipeline_builder.build()
+
+        sdc_executor.add_pipeline(pipeline)
+        snapshot = sdc_executor.capture_snapshot(pipeline, start_pipeline=True, batch_size=3).snapshot
+        item_list = [output.field for output in snapshot[directory.instance_name].output]
+        rows_from_snapshot = [{item['title'][0]['value'].value: item['prc:price'][0]['value'].value}
+                              for item in item_list]
+        # Parse input xml data to verify results from snapshot using xpath for search.
+        root = ElementTree.fromstring(FILE_CONTENTS)
+        expected_data = [{msg.find('title').text: msg.find('{http://books.com/price}price').text}
+                         for msg in root.findall('{http://books.com/book}book')]
+        assert rows_from_snapshot == expected_data
+
+        if include_field_xpaths:
+            output_records = snapshot[directory.instance_name].output
+            # Test for Record Headers
+            record_header = [record.header.values for record in output_records]
+            assert record_header[0]['xmlns:b'] == 'http://books.com/book'
+            assert record_header[0]['xmlns:prc'] == 'http://books.com/price'
+            # Test for Field Headers .Currently using _data property since api for filed header is not there.
+            field_info = output_records[0]._data['value']['value']
+            assert field_info['title']['value'][0]['value']['attr|lang']['attributes'][
+                       'xpath'] == '/bookstore/b:book/title/@lang'
+            assert field_info['title']['value'][0]['value']['value']['attributes']['xpath'] == '/bookstore/b:book/title'
+            assert field_info['prc:price']['value'][0]['value']['value']['attributes'][
```

Comment:
Probably use `()` to avoid splitting dictionary like this..
```suggestion
            assert (field_info['prc:price']['value'][0]['value']['value']['attributes']['xpath'] == 
                         '/bookstore/b:book/prc:price')
```